docs/content/en/blog/releases/v0.37.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Release v0.37.0"
+linkTitle: "Release v0.37.0"
+date: 2022-09-07
+description: >
+ Release v0.37.0
+---
+
+## Changes since v0.36.0
+
+> Starting from version v0.37.0, it's able to control the order of applying Kubernetes manifests with `pipecd.dev/order` annotation.\
+  In this example, CRD manifest will be applied before being applied Deployment manifest.
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: sample
+  annotations:
+    pipecd.dev/order: 1
+spec:
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sample
+  annotations:
+    pipecd.dev/order: 2
+spec:
+```
+
+### Notable Changes
+* Allow using Annotations to sort Kubernetes manifests before applying ([#3869](https://github.com/pipe-cd/pipecd/pull/3869))
+* Replace old RBAC handling with new RBAC handling ([#3868](https://github.com/pipe-cd/pipecd/pull/3868))
```

Comment:
instead of making this suspicious note a notable change without explanation, how about adding a more stand out quote (like above for the k8s new feature) such as: start from this version, PipeCD changes its internal requests permission handling mechanism, operators of the self-hosted PipeCD control plane are required to update the control plane to this version at least, the users will only have to re-login to the control plane after the update.
wdyt?

Reply:
> I guess operators and users don't need to do specific things because if control-plane's version is under this version, it's okay to contact any resources and if users didn't refresh their tokens, GetMe() will be failed, and then redirect to login pages.

Nice point, then how about making this internal change instead? 🤔

Reply:
Nice idea!
I addressed it by https://github.com/pipe-cd/pipecd/pull/3899/commits/6b82c77c9ee4328582429c295953d010e8823020.